ui/src/routes/admin/users/create-user/index.tsx

```diff
@@ -56,42 +65,85 @@ const formSchema = z.object({
   email: asOptionalField(z.string().email()),
   password: asOptionalField(z.string().min(1)),
   temporary: z.boolean(),
+  organizations: z.array(z.string()).min(1, {
+    message:
+      'You must grant the user READERS access to at least one organization',
+  }),
 });
 
 const CreateUser = () => {
   const navigate = useNavigate();
 
-  const { mutateAsync, isPending } = useAdminServicePostApiV1AdminUsers({
-    onSuccess() {
-      toast.info('Add User', {
-        description: `User "${form.getValues().username}" added successfully to the server.`,
-      });
-      navigate({
-        to: '/admin/users',
-      });
-    },
-    onError(error: ApiError) {
-      toast.error(error.message, {
-        description: <ToastError error={error} />,
-        duration: Infinity,
-        cancel: {
-          label: 'Dismiss',
-          onClick: () => {},
-        },
-      });
-    },
+  const {
+    data: organizations,
+    isPending: orgIsPending,
+    isError: orgIsError,
+    error: orgError,
+  } = useOrganizationsServiceGetApiV1Organizations({
+    limit: ALL_ITEMS,
   });
 
+  const { mutateAsync: createUser, isPending: isCreateUserPending } =
+    useAdminServicePostApiV1AdminUsers({
+      onSuccess() {
+        toast.info('Add User', {
+          description: `User "${form.getValues().username}" added successfully to the server.`,
+        });
+      },
+      onError(error: ApiError) {
+        toast.error(error.message, {
+          description: <ToastError error={error} />,
+          duration: Infinity,
+          cancel: {
+            label: 'Dismiss',
+            onClick: () => {},
+          },
+        });
+      },
+    });
+
+  const {
+    mutateAsync: addUserToReaders,
+    isPending: isAddUserToReadersPending,
+  } =
+    useOrganizationsServicePutApiV1OrganizationsByOrganizationIdGroupsByGroupId(
+      {
+        onSuccess(_, variables) {
+          const organizationName = organizations?.data.find(
+            (org) => org.id.toString() === variables.organizationId.toString()
+          )?.name;
+
+          toast.info('Add User', {
+            description: `User "${variables.requestBody?.username}" granted READERS access to "${organizationName}".`,
```

Comment:
I propose

   `The "${variables.requestBody?.username}" user was created and added to the "${organizationName}" organization as part of the READERS group.`